### DISCUSSION TITLE
does clickhouse has a Statement cache? how can i configure it?

### DISCUSSION THREAD

**Discussion by Qinyingfeng** [ClickHouse/ClickHouse#51176] [Q&A] [answered]
Most major database JDBC drivers already have a Statement cache that can be configured, including PostgreSQL, Oracle, Derby, MySQL, DB2, and many others.
so my question is : does clickhouse has a Statement cache? how can i configure it?

if clickhouse cache prestatement, so that I can get the cached prestatement in different connections instead of creating a new prestatement every time, because creating a new prestatement may be expensive

**Comment by den-crane** [comment]
https://clickhouse.com/docs/en/operations/query-cache

**Comment by den-crane** [comment, marked as answer]
Clickhouse does not have prepared statements. No such feature.
